Cargo.toml

```diff
@@ -27,5 +27,14 @@ name = "tree-sitter-grep"
 
 [dev-dependencies]
 assert_cmd = "2.0.11"
+escargot = "0.5.7"
 predicates = "3.0.3"
 shlex = "1.1.0"
+
+[[example]]
+name = "filter_before_line_number"
+crate-type = ["cdylib"]
```

Comment:
Per the [docs](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples), specify that these examples are shared libraries

Reply:
Interesting, I wouldn't have guessed that a dynamic library was a valid type of "example".

Reply:
Ya the author of [this article](http://xion.io/post/code/rust-examples.html#fn:1) explicitly says he thinks that isn't useful but _boy was he wrong_